Update Ledger API version and its documentation (#14986)
digital-asset/daml: docs/source/support/compatibility.rst

   * - Ledger API Version
     - Changes
   * - 2.1
     - | Establish the order of child events in ExercisedEvent to agree with the order of events in transaction.
       | Indicate an exercise done on an interface through the interface_id field on the ExercisedEvent message.

>        | Make interfaces available for subscriptions in the Transaction Service as a preview feature.

Reviewer: ```suggestion
       | Make interfaces available for subscriptions in the Transaction Service as an Alpha feature.
```

Author: :+1:

>        | Implement contract disclosure as an experimental feature in the Transaction, Command Submission and Command Services. Related new message types of DisclosedContract and ContractMetadata should be ignored.

Reviewer: ```suggestion
       | Implement contract disclosure as a Labs feature in the Transaction, Command Submission and Command Services. Related new message types of DisclosedContract and ContractMetadata should be ignored.
```

Author: :+1:
